src/rules/selector-type-no-unknown/index.js

```diff
@@ -18,6 +18,32 @@ export const messages = ruleMessages(ruleName, {
   rejected: (selector) => `Unexpected unknown type selector "${selector}"`,
 })
 
+// htmlTags includes only "standard" tags. So we augment it with older tags etc.
+const nonStandardHtmlTags = new Set([
```

Comment:
Are these all "standard" but *obsolete*? If so, `obsoleteHtmlTags` would be a more accurate name.